zipkin/src/main/scala/org/http4s/zipkin/interpreters/clock/package.scala

```diff
@@ -0,0 +1,12 @@
+package org.http4s.zipkin.interpreters
+
+import java.time.Instant
+
+import org.http4s.zipkin.algebras.Clock
+
+package object clock {
```

Comment:
Focus is currently on Zipkin, but nothing about a pure clock or pure randomness relates to Zipkin, and these concepts could potentially be used elsewhere to make things more testable in a deterministic fashion.   Possible move into core util.